lisa/features/security_profile.py

```diff
@@ -59,6 +59,12 @@ class SecurityProfileSettings(schema.FeatureSettings):
         ),
     )
     encrypt_disk: bool = field(default=False)
+    disk_encryption_set_id: str = field(
```

Comment:
If it's an Azure specified setting, it should be defined in Azure's feature settings.

Reply:
I'm having trouble moving this field. _call_requirement_method() needs access to the field but moving the method to Azure also doesn't seem to work. Please advise.

Reply:
Azure should inherit this method, and do extra logic after called the super one.

Reply:
Fixed